packages/components/src/IconItem/index.tsx

```diff
@@ -16,13 +16,16 @@ export interface InterfaceIconItem {
     | "secondary"
     | "danger"
     | "warning"
-    | "success";
+    | "success"
+    | "blue-light";
```

Comment:
Ídem, acá cambiaría a `"secondary-light"`

```suggestion
    | "secondary-light";
```

Reply:
bien, también cambiado